### PR TITLE
Merge update actions

### PR DESCRIPTION
**Pull request by dcermak**
#1325 showed a flaw in our auto-update logic: if the mariadb version is bumped, then everything burns down in flames, as the necessary files are no longer present. To fix that, we **must** fetch the new files in the same PR to have a seamless update UX

### CODE REVIEW

**Review by alexandrevicenzi — DISMISSED** [review]

File: .github/workflows/update-versions-and-files.yml [hunk 1]
```diff
@@ -28,17 +28,23 @@ jobs:
           OSC_PASSWORD: ${{ secrets.OSC_PASSWORD }}
           OSC_USER: "defolos"
 
-      - name: create a pull request with the updated versions
+      # update the files **after** we updated the versions to get the new
+      # mariadb files
```
Comment (dirkmueller) [inline .github/workflows/update-versions-and-files.yml:32]: this comment is hard to parse. how about

```suggestion
# update versions first so that updating the files  can fetch based on the updated versions
```

File: .github/workflows/update-versions-and-files.yml [hunk 2]
```diff
@@ -28,17 +28,23 @@ jobs:
...
+          title: "🤖: Update package versions and files"
           body: |
-            This is an automated update via poetry run update-versions
+            This is an automated update via poetry run update-versions and ./update-files.sh
           base: main
-          branch: create-pull-request/update-versions
+          branch: create-pull-request/update-versions-and-files
           add-paths: |
-            src/bci_build/package/package_versions.json
+            src/bci_build/package/*
```
Comment (dirkmueller) [inline .github/workflows/update-versions-and-files.yml:50]: eh, does this work recursively?

Comment (dcermak) [reply .github/workflows/update-versions-and-files.yml:50]: Yes, see the deleted `.github/workflows/update-scripts.yml`

**Review by dirkmueller — APPROVED** [review]
